testsuite/MDAnalysisTests/analysis/test_dssp.py

```diff
@@ -27,6 +27,14 @@ def test_trajectory():
 
     assert first_frame[:10] != last_frame[:10] == avg_frame[:10] == '-EEEEEE---'
 
+def test_atomgroup():
+    u = mda.Universe(TPR, XTC).select_atoms('protein')
+    run = DSSP(u).run(stop=10)
```

Comment:
Make the code clearer — using `u` for an AtomGroup confuses the hell out of anyone writing code with MDA:
```suggestion
    protein = mda.Universe(TPR, XTC).select_atoms('protein')
    run = DSSP(protein).run(stop=10)
```